camera:
  distortion_coefficients:  [-0.108400, 0.141934, -0.000856, 0.004049, 0.000000]
  intrinsic_parameters: [548.061325, 0.000000, 306.931549, 0.000000, 546.563662, 218.531695, 0.000000, 0.000000, 1.000000]
box:
  minSize: 400
  minAreaIndex: 0.2
flags:
  strictCheck: false
  rqt: true
  debug: false
